client/src/components/shared/Info/RelationList.js

```diff
@@ -38,14 +38,22 @@ const RelationItem = ({related, name, useNewApi}: RelationItemProps) => (
   </li>
 )
 
-const TitleBadge = ({related, name, useNewApi}: RelationItemProps) =>
-  related.edge_types.map((type: number, i: number) => (
-    <Badge
-      key={type}
-      color={type > 0 ? 'dark' : 'secondary'}
-      title={getRelationTitle(type, name, related.name)}
-      className="mr-1 info-badge"
-    >
-      {showRelationType(type, related.edge_type_texts[i])}
-    </Badge>
-  ))
+const TitleBadge = ({related, name, useNewApi}: RelationItemProps) => {
```

Comment:
Not your code, but please remove useNewApi prop, as it is not used here, and also from line 36. You might need to create a new proptype, so that flow does not see it as an error.